File: go/1030.A.go
package main

import (
	"fmt"
)

func main() {

	var n, b int
	fmt.Scan(&n)
	for i := 0; i < n; i++ {
		fmt.Scan(&b)
		if b == 1 {
			fmt.Println("HARD")
			return
		}
	}
	fmt.Println("EASY")

}

File: dotNet/1030.A.cs
int n = int.Parse(Console.ReadLine());
string[] s = Console.ReadLine().Split(' ');

int r = Array.ConvertAll(s, int.Parse).Sum();
if (r == 0)
{
    Console.WriteLine("EASY");
}
else {
    Console.WriteLine("HARD");
}
